src/components/shared/chart/Canvas.js

```diff
@@ -111,6 +116,7 @@ export default class ChartCanvas<HoveredItem> extends React.Component<
       return;
     }
 
+    this._mouseMoved = true;
```

Comment:
I think this is the good approach but a bit too crude: the problem is that some people may move the mouse a bit while "clicking", and so for accessibility this can be an issue.
So I think we need to introduce a "delta from the mouseDown" to make sure we moved more than eg 5 pixels in one direction. This means that in `mousedown` we should store the start point, and compare it in `mousemove` to define the `mousemoved` boolean state.

Also please add somewhere a comment explaining that we can't use `click` because that event is fired even if the cursor moved a lot.